youtube_dl/extractor/franceculture.py

```diff
@@ -36,12 +36,12 @@ def _real_extract(self, url):
                     </h1>|
                     <div[^>]+class="[^"]*?(?:title-zone-diffusion|heading-zone-(?:wrapper|player-button))[^"]*?"[^>]*>
                 ).*?
-                (<button[^>]+data-asset-source="[^"]+"[^>]+>)
+                (<button[^>]+data-(asset-source|url)="[^"]+"[^>]+>)
             ''',
             webpage, 'video data'))
 
-        video_url = video_data['data-asset-source']
-        title = video_data.get('data-asset-title') or self._og_search_title(webpage)
+        video_url = video_data.get('data-asset-source') or video_data.get('data-url')
```

Comment:
Thanks for your help, that's my first contribution to this project and I'm still a bit lost after reading the coding conventions.

As far as I know, either `data-asset-source` or `data-url` can be found in the HTML, on the previous and current version of the FranceCulture website, respectively. This is the root cause of the current extractor being broken, and the very reason of this PR. If both are missing, though, `video_url` will be `False` and I understand that's inappropriate and the need for a default value. Yet, I can't figure out what to use since none seem to make sense as the extractor will fail anyway.

What would be an acceptable solution?